HousingManagementSystemApi/Properties/launchSettings.json

```diff
@@ -27,7 +27,9 @@
         "ASPNETCORE_ENVIRONMENT": "Development",
         "AUTHENTICATION_IDENTIFIER": "<REPLACE WITH A UNIQUE AUTHENTICATION IDENTIFIER>",
         "JWT_SECRET" : "<REPLACE WITH A SECRET>",
-        "SENTRY_DNS": "https://<REPLACE WITH SENTRY DSN>"
+        "SENTRY_DNS": "https://<REPLACE WITH SENTRY DSN>",
+        "HOUSING_SEARCH_API_URI":"https://<REPLACE WITH API URI>",
+        "HOUSING_SEARCH_API_KEY":"https://<REPLACE WITH API KEY>"
```

Comment:
Do we need to add this for the properties uri and key

Reply:
Added